godel_plugins/include/godel_plugins/widgets/robot_blending_widget.h

```diff
@@ -61,12 +62,18 @@ const std::string GET_AVAILABLE_MOTION_PLANS_SERVICE = "get_available_motion_pla
 const std::string SELECT_MOTION_PLAN_SERVICE = "select_motion_plan";
 const std::string LOAD_SAVE_MOTION_PLAN_SERVICE = "load_save_motion_plan";
 const std::string RENAME_SURFACE_SERVICE = "rename_surface";
+const static std::string PROCESS_PLANNING_ACTION_SERVER_NAME = "process_planning_as";
 
 class RobotBlendingWidget : public QWidget
 {
   Q_OBJECT
 public:
-  RobotBlendingWidget(std::string ns = "");
+  RobotBlendingWidget(std::string ns = "") :
+    param_ns_(ns), process_planning_action_client_(PROCESS_PLANNING_ACTION_SERVER_NAME, true)
```

Comment:
Does this action client needs its own thread (the `true` argument to the second parameter)?